templates/bot.mustache

```diff
@@ -1,168 +1,164 @@
+                        <div class="topMenuBalance">0</div>
+                      </div>
+                      <div class="nav-caret"></div>
+                    </a>
+                    <ul>
+                      <div class="login-id-list"></div>
+                      <a href="javascript:;" id="btn_logout" class="logout">
+                        <li id="logout" data-i18n-text="Log out"></li>
+                      </a>
+                    </ul>
+                  </li>
                 </ul>
+              </div>
             </div>
-          </div>
-          <div>
-            <a class="button">
+            <div>
+              <a class="button">
                 <span id="login" data-i18n-text="Log in"></span>
-            </a>
+              </a>
+            </div>
           </div>
         </div>
       </div>
-  </div>
-  <div id="blocklyDiv"></div>
-  <div id="summaryPanel" class="panel">
+    </div>
+    <div id="blocklyDiv"></div>
+    <div id="summaryPanel" class="panel">
       <div class="panelExitButton"><p class="icon-close"></p></div>
       <div class="description">
-          <h4 data-i18n-text="Results"></h4>
+        <h4 data-i18n-text="Results"></h4>
       </div>
-    <div class="content">
-      <h4>
+      <div class="content">
+        <h4>
         <span data-i18n-text="Summary"></span>
-      </h4>
-      <table id="summaryDisplay">
-        <thead>
-          <tr>
-            <th data-i18n-text="No. of runs"></th>
-            <th data-i18n-text="Total Stake"></th>
-            <th data-i18n-text="Total Payout"></th>
-            <th data-i18n-text="Win"></th>
-            <th data-i18n-text="Loss"></th>
-            <th data-i18n-text="Total Profit/Loss"></th>
-            <th data-i18n-text="Balance"></th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr>
-            <td class="totalRuns"></td>
-            <td class="totalStake"></td>
-            <td class="totalPayout"></td>
-            <td class="totalWins"></td>
-            <td class="totalLosses"></td>
-            <td class="totalProfit"></td>
-            <td class="balance"></td>
-          </tr>
-        </tbody>
-      </table>
-      <h4>
-        <span data-i18n-text="Trades"></span>
-      </h4>
-      <div class="table-scroll">
-        <table id="tradesDisplay">
+        </h4>
+        <table id="summaryDisplay">
           <thead>
             <tr>
-              <th><span data-i18n-text="Number"></span></th>
-              <th><span data-i18n-text="Reference"></span></th>
-              <th><span data-i18n-text="Trade Type"></span></th>
-              <th><span data-i18n-text="Entry Spot"></span></th>
-              <th><span data-i18n-text="Exit Spot"></span></th>
-              <th><span data-i18n-text="Buy Price"></span></th>
-              <th><span data-i18n-text="Final Price"></span></th>
-              <th><span data-i18n-text="Profit/Loss"></span></th>
+              <th data-i18n-text="No. of runs"></th>
```

Comment:
i didn't add this tho.. it was already there, probably fetched from the beta branch